plug.vim

```diff
@@ -775,6 +776,18 @@ function! s:update_impl(pull, force, args) abort
     let s:py = s:version_requirement(
           \ map(split(split(pyv)[0], '\.'), 'str2nr(v:val)'), [2, 6])
   endif
+
+  " TODO: Remove when jobwait() blocks correctly
+  if has('nvim') && has('vim_starting')
+    if s:py
+      let s:nvim = 0
+    else
+      echohl WarningMsg
+      echomsg 'vim-plug: +PlugInstall/+PlugUpdate synchronous only with python support'
+      echomsg 'vim-plug: pip(3) install neovim'
+      echohl None
+    endif
+  endif
```

Comment:
https://github.com/junegunn/vim-plug/issues/104#issuecomment-98295463

`vim +PlugUpdate` will cause any subsequent install/update to become synchronous. I'd prefer to remove `s:` variables like I did in https://github.com/junegunn/vim-plug/issues/104#issuecomment-98295463.